observability-kit-starter/pom.xml

```diff
@@ -41,6 +39,10 @@
             <version>4.7.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-sdk</artifactId>
+        </dependency>
```

Comment:
Dependency can perhaps be removed, as we do not have any more direct references to open telemetry API
```suggestion
```